integration-tests/tests/register-user.js

```diff
@@ -34,8 +46,12 @@ const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
 
   // Accept the Terms of Service
   await click(page, clickable({ textContains: 'Read Terra Platform Terms of Service here' }));
-  await click(page, clickable({ textContains: 'OK' }));
-  await click(page, clickable({ textContains: 'By checking this box, you are agreeing to the Terra Terms of Service' }));
+  await waitForModal(page);
+  // We seem to need a small delay here
```

Comment:
What happens without the delay?

Reply:
The test fails to find the OK button and fails.

Reply:
ah, ok. `waitForNoSpinners` works here too.